src/Tools/dotnet-watch/src/dotnet-watch.csproj

```diff
@@ -22,13 +22,42 @@
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore" />
     <Reference Include="Microsoft.AspNetCore.WebSockets" />
+
+    <ProjectReference
+      Include="..\BrowserRefresh\src\Microsoft.AspNetCore.Watch.BrowserRefresh.csproj"
+      PrivateAssets="All"
+      ReferenceOutputAssembly="false"
+      SkipGetTargetFrameworkProperties="true"
+      UndefineProperties="TargetFramework;TargetFrameworks" />
```

Comment:
Not now (because the metadata won't be handled correctly) but when this is merging into 'master', switch to use a `@(Reference)` item instead. We're avoiding direct project references as much as possible.